src/collection-view.js

```diff
@@ -30,6 +26,11 @@ Marionette.CollectionView = Marionette.View.extend({
     Marionette.View.apply(this, arguments);
 
     this.on('show', this._onShowCalled);
+
+    if (_.isUndefined(this.sort)) {
+        _.defaults(this.options, { sort: true });
+    }
```

Comment:
So a couple thoughts. 
1) this boilerplate rubs me the wrong way
2) I think sort: true as a prototype property would work, subclasses could override in their proto or initialize
3) getOption should take a default as a last param. It's been discussed.